MAINTAIN.md

```diff
@@ -160,7 +160,6 @@ These dependencies are "vendored" (inlined), we must update the sources manually
     * Needs to be updated when LPeg is updated.
 * `src/bit.c`: only for PUC lua: port of `require'bit'` from luajit https://bitop.luajit.org/
 * `runtime/lua/coxpcall.lua`: coxpcall (only needed for PUC lua, builtin to luajit)
-* `src/termkey`: [libtermkey](https://github.com/neovim/libtermkey)
```

Comment:
why not list it here still? even if the upstream is frozen, we still vendored it and it's useful to have a reference list of all the things we vendored. we still list unibilium too.

Reply:
Unibilium is not vendered

Reply:
> is no longer true: the sources now live in-tree; there will no longer be an external repo from which we need to pull updates. (The repo is already archived.)

That was pretty much my thought process. It's not something I'm for or against either way so I don't mind re-adding it.